### PR TITLE
Update GraphQL version

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -3,7 +3,7 @@ import sbt._
 object Dependencies {
   lazy val awsUtils =  "uk.gov.nationalarchives.aws.utils" %% "tdr-aws-utils" % "0.1.15"
   lazy val catsEffect = "org.typelevel" %% "cats-effect" % "2.2.0"
-  lazy val generatedGraphql = "uk.gov.nationalarchives" %% "tdr-generated-graphql" % "0.0.173"
+  lazy val generatedGraphql = "uk.gov.nationalarchives" %% "tdr-generated-graphql" % "0.0.187"
```
Comment (TomJKing) [inline project/Dependencies.scala:6]: Looks like there are some duplicated library versions. For example the log4cats' libraries. Might be worth pulling these out into variables.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one very minor comment.

File: project/Dependencies.scala (hunk 1 above)
Comment (techncl) [reply project/Dependencies.scala:6]: Hi Tom, but these libraries are different; do you still want me to move their versions into a variable?

Comment (TomJKing) [reply project/Dependencies.scala:6]: Fair point.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**